### PR TITLE
add arabic localization

### PR DESCRIPTION
**Pull request by AbdelhamidGamal**
arabic localization

### CODE REVIEW

File: app/src/main/res/values-ar/about.xml [hunk 1]
```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="about_close">Close</string>
```
Comment (erciccione) [inline app/src/main/res/values-ar/about.xml]: This and the following two strings can be translated

**Review by erciccione — COMMENTED** [review]
@3b7ameed some last minute fixes i missed earlier. Sorry about that

File: app/src/main/res/values-ar/about.xml (hunk 1 above)
Comment (AbdelhamidGamal) [reply app/src/main/res/values-ar/about.xml]: Done

**Review by erciccione — APPROVED** [review]

**Comment by m2049r** [comment]
there are a couple of layout changes i need to make before merging this.

question: the numberpad layout of digits is the same in arabic like US?

**Comment by AbdelhamidGamal** [comment]
> there are a couple of layout changes i need to make before merging this.
> 
> question: the numberpad layout of digits is the same in arabic like US?

yes it's the same

**Pull request closed without merging**